SHOWCASE.md

```diff
@@ -2,67 +2,85 @@
-| [MusicBrainz Songs Search](https://songs-search.typesense.org/) | A site that showcases Typesense in action on a 32 Million Songs database from [MusicBrainz](https://musicbrainz.org/) |
-| [2020 US Presidential Candidates' Speeches Search](https://biden-trump-speeches-search.typesense.org/) | Instantly Search speeches of US Presidential Candidates side-by-side. |
-| [E-commerce Store Browsing Experience](https://ecommerce-store.typesense.org/) | A site that showcases how to build an e-commerce storefront browsing experience with Typesense. |
+| Name                                                                                                   | Description                                                                                                                                                               |
+| ------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Recipe Search](https://recipe-search.typesense.org/)                                                  | A site that showcases Typesense in action on a 2 Million recipe database, with the ability to filter by ingredients.                                                      |
+| [Linux Commit History Search](https://linux-commits-search.typesense.org/)                             | A site that indexes 1M linux commit messages in Typesense and lets you browse, search and filter through the commits.                                                     |
+| [Books Search](https://books-search.typesense.org/)                                                    | A site that showcases Typesense in action on a 28 Million books database from [OpenLibrary](https://openlibrary.org/), with the ability to filter by authors and subject. |
+| [MusicBrainz Songs Search](https://songs-search.typesense.org/)                                        | A site that showcases Typesense in action on a 32 Million Songs database from [MusicBrainz](https://musicbrainz.org/)                                                     |
+| [2020 US Presidential Candidates' Speeches Search](https://biden-trump-speeches-search.typesense.org/) | Instantly Search speeches of US Presidential Candidates side-by-side.                                                                                                     |
+| [E-commerce Store Browsing Experience](https://ecommerce-store.typesense.org/)                         | A site that showcases how to build an e-commerce storefront browsing experience with Typesense.                                                                           |
```

Comment:
@tharropoulos Could you also add all the new demos we now have from this page: https://typesense.org/docs/overview/demos.html

Reply:
>@tharropoulos Could you also add all the new demos we now have from this page: https://typesense.org/docs/overview/demos.html

Addressed in c30f6230c0e7bdd2220b5c576aacc2f51e7726fe